frontend/packages/noobaa-storage-plugin/src/plugin.ts

```diff
@@ -131,6 +147,78 @@ const plugin: Plugin<ConsumedExtensions> = [
       required: NOOBAA_FLAG,
     },
   },
+  {
+    type: 'NavItem/ResourceCluster',
+    properties: {
+      section: 'Storage',
+      componentProps: {
+        name: 'Object Buckets',
+        resource: models.NooBaaObjectBucketModel.plural,
+        required: NOOBAA_FLAG,
+      },
+    },
+  },
+  {
+    type: 'Page/Resource/List',
+    properties: {
+      model: models.NooBaaObjectBucketModel,
+      loader: () =>
+        import(
+          './components/object-bucket-page/object-bucket' /* webpackChunkName: "object-bucket-page" */
+        ).then((m) => m.ObjectBucketsPage),
+    },
+  },
+  {
+    type: 'Page/Resource/Details',
+    properties: {
+      model: models.NooBaaObjectBucketModel,
+      loader: () =>
+        import(
+          './components/object-bucket-page/object-bucket' /* webpackChunkName: "object-bucket-page" */
+        ).then((m) => m.ObjectBucketDetailsPage),
+    },
+  },
+  {
+    type: 'NavItem/ResourceNS',
+    properties: {
+      section: 'Storage',
+      componentProps: {
+        name: 'Object Bucket Claims',
+        resource: models.NooBaaObjectBucketClaimModel.plural,
+        required: NOOBAA_FLAG,
+      },
+    },
+  },
+  {
+    type: 'Page/Resource/List',
+    properties: {
+      model: models.NooBaaObjectBucketClaimModel,
+      loader: () =>
+        import(
+          './components/object-bucket-claim-page/object-bucket-claim' /* webpackChunkName: "object-bucket-claim-page" */
+        ).then((m) => m.ObjectBucketClaimsPage),
+    },
+  },
+  {
+    type: 'Page/Resource/Details',
+    properties: {
+      model: models.NooBaaObjectBucketClaimModel,
+      loader: () =>
+        import(
+          './components/object-bucket-claim-page/object-bucket-claim' /* webpackChunkName: "object-bucket-claim-page" */
+        ).then((m) => m.ObjectBucketClaimsDetailsPage),
+    },
+  },
+  {
+    type: 'Page/Route',
+    properties: {
+      path: `/k8s/ns/:ns/objectbucketclaims/~new/form`,
```

Comment:
since OBC is CRD, I think you should use `referenceForModel(NooBaaObjectBucketClaimModel)`

Reply:
ack